cmd/release.go

```diff
@@ -64,14 +74,29 @@ func uploadTarball(path string, f os.FileInfo, err error) error {
 	}
 
 	if matched {
-		sh("github-release upload",
-			"--user", info.Owner,
-			"--repo", info.Name,
-			"--tag", fmt.Sprintf("v%s", info.Version),
-			"--name", fileName,
-			"--file", path)
+		err := retry.Do(func(attempt int) (bool, error) {
+			var err error
+			err = uploadTarball(fileName, path)
+			if err != nil {
+				time.Sleep(2 * time.Second)
+			}
+			return attempt < allowedRetry+1, err
```

Comment:
That doesn't make much sense to me TBH. If a user supplies a too large retry mechanism, either just allow that or shout a warning/error in their direction immediately. But I don't think there should be two places that check the number of retries with different logic.